src/OpenIddict.Validation/OpenIddictValidationHandlers.cs

```diff
@@ -17,9 +17,9 @@ public static partial class OpenIddictValidationHandlers
         /*
          * Authentication processing:
          */
-        ResolveServerConfiguration.Descriptor,
         EvaluateValidatedTokens.Descriptor,
         ValidateRequiredTokens.Descriptor,
+        ResolveServerConfiguration.Descriptor,
```

Comment:
Can you please also move the handler code so that it appears at the same place in the file as defined here?